sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -373,16 +377,19 @@ export default {
         await formioUtils.resetCheckBox(form, PROGRAM_NOT_LISTED, {
           programnotListed: false,
         });
+        const selectedLocationId = getSelectedId(form);
 
-        // when isReadOnly.value is true, then consider
-        // both active and inactive program year.
-        await formioDataLoader.loadProgramsForLocation(
-          form,
-          +event.changed.value,
-          PROGRAMS_DROPDOWN_KEY,
-          props.programYearId,
-          isReadOnly.value,
-        );
+        if (selectedLocationId) {
+          // when isReadOnly.value is true, then consider
+          // both active and inactive program year.
+          await formioDataLoader.loadProgramsForLocation(
+            form,
+            +selectedLocationId,
+            PROGRAMS_DROPDOWN_KEY,
+            props.programYearId,
+            isReadOnly.value,
```

Comment:
If we are going to do label for selected program, we can ignore this part right. Correct me if I am missing something.

Reply:
`label` is not part of these PR. and this logic or codes are not added by me. It already, I only added some conditions and some different logic to resolve the program intensity refreshing bug in the application form, 
![image](https://user-images.githubusercontent.com/77353155/155209019-8530a151-03f2-4165-8f50-9b85bb2f225a.png)